sdk/ai/azure-ai-generative/azure/ai/generative/evaluate/_metric_handler.py

```diff
@@ -112,7 +112,17 @@ def calculate_metrics(self) -> Dict:
         result_metrics = pf_client.get_metrics(pf_run.name)
 
         # Drop unselected output columns
-        columns_to_drop = [col for col in result_df.columns if col.replace("outputs.", "") not in metrics]
+        #columns_to_drop = [col for col in result_df.columns if col.replace("outputs.", "").replace("_reasoning", "").replace("_score", "") not in metrics]
+        columns_to_drop = []
```

Comment:
Thx for adding this. Can you add a quick comment to explain what columns will be dropped?

Reply:
any columns whose column name after removing "outputs." does not start with any selected metric will be dropped. e.g. if the metrics are ["gpt_groundedness", "self_harm"]. if the column name after removing "outputs." does not start with "gpt_groundedness" or ""self_harm" will be dropped. the columns to be kept are ["gpt_groundedness", "self_harm", "self_harm_score", "self_harm_reasoning"]